1.1/nanoserver-sac2016/sdk/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # escape=`
-FROM mcr.microsoft.com/dotnet/core/sdk:1.1.12-nanoserver-sac2016
+FROM mcr.microsoft.com/dotnet/core/sdk:1.1.13-nanoserver-sac2016
```

Comment:
The nanoserver-sac2016 images were deprecated from the .NET Core repos and therefore aren't being serviced any longer - https://github.com/dotnet/dotnet-docker/pull/957.

Reply:
@JunTaoLuo Let's skip the images on our end as well (the ones that are based on`nanoserver-sac2016`).  Usage of these are extremely low, and we don't have a supported base image anyway.

Reply:
Sounds good.

Reply:
I removed all nanoserver-sac2016 images and sections in readmes. Is that all we need to do?